tests/integration_tests/conftest.py

```diff
@@ -176,6 +196,53 @@ async def token(request):
     await runner.cleanup()
 
 
+@pytest.fixture
+async def jwks_server():
+    host = "localhost"
+    port = 4080
+
+    app = web.Application()
+    routes = web.RouteTableDef()
+
+    class Handle(object):
+        def __init__(self):
+            self.host = host
+            self.port = port
+            self.result = json.loads(textwrap.dedent("""
+            {"keys":[
+            {
```

Comment:
No need to call `loads` here as JSON is also syntactically valid python.
```suggestion
            self.result = {
                "keys": [{
```